torch/fx/experimental/proxy_tensor.py

```diff
@@ -303,6 +304,30 @@ def __torch_dispatch__(cls, func_overload, types, args=(), kwargs=None):
                     break
         assert proxy_mode is not None, "At least one argument must be a ProxyTensor"
 
+        if func_overload.__name__ == 'torch.cond':
+            # TODO: need to parse
+            print("In cond")
+            assert kwargs is None or not kwargs
+            pred, true_fn, false_fn, *operands = args
+            true_graph = get_isolated_graphmodule(true_fn, *operands, {})
```

Comment:
MyPy correctly complaints here, that the arguments to this function are: `func, args, kwargs`. So `operands` should be passed here without unpacking.

Reply:
Alright, no problem. Fwiw this part of the pr works fine ;) we are still having issues on the proxy tensor capturing a bool value specifically